docs/src/wallet-guide/apps.md

```diff
@@ -5,6 +5,16 @@ title: Mobile App Wallets
 Solana is supported by multiple third-party apps which should provide a familiar
 experience for most people who are new or experienced with using crypto wallets.
 
+## Solflare
```

Comment:
Do you mind alphabetizing the mobile app wallets?

Reply:
@jstarry is that really a good approach? Same as web wallets, my guess was to sort it by available features, "relevance" and popularity. I'm guessing Solana-first wallet with this mass adoption deserves to be above wallet that doesn't promote decentralisation by forcing all stakes to one validator. I guess Phantom will probably go above Solflare when they release mobile app as well because they are more popular but offer similar functionalities.

Reply:
I think alphabetical is better for now, I don't want to maintain an ordering of popularity

Reply:
We absolutely _do not_ want to consider "popularity" or "relevance" here.  These are documents, not ad space

Reply:
thanks @jstarry @t-nelson I got it! Do you want me to sort web wallets alphabetically as part of this PR as well then?